index.bs

```diff
@@ -509,7 +509,7 @@ spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
   The `PortalHost` interface {#the-portalhost-interface}
   ------------------------------------------------------
 
-  The <dfn>portal host object</dfn> of a [=browsing context=] is a {{PortalHost}}.
+  The <dfn>portal host object</dfn> of a {{Window}} is a {{PortalHost}}.
```

Comment:
Noticed an inconsistency about which object has the portal host object. It should be the Window, since the platform object should be different across navigations, even if the new one's operations will ultimately do the same thing. (Else the thing will leak across realms etc.)